bin/ert.in

```diff
@@ -33,41 +21,42 @@ def ensureConfig(func):
 
 
 def runExec(executable , args):
-    exec_args = [ executable ] + args
-    os.execvpe( executable , exec_args , os.environ )
-
+    os.execvp(executable, [executable] + args)
 
 
 @ensureConfig
 def runGui( args ):
-    runExec("python" , ["%s/ert_gui/gert_main.py" % ERT_PYTHONPATH] + args)
+    runExec("python", ["-m", "ert_gui.gert_main"] + args)
```

Comment:
NB: Totally unrelated to this PR; but this PR might still be the place to check/fix it:
```bash
prompt% ert --gui debug.ert
usage: gert_main.py [-h] [-d] obs_file contact_result_file
gert_main.py: error: too few arguments
```
i.e. - when the option `--gui` is passed there is an error message. The `--gui` option is anyways the default behavior, and not much used.